spec/migrations/20170317153953_copy_agent_id_to_miq_server_id_in_jobs_table_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe CopyAgentIdToMiqServerIdInJobsTable do
+  let(:jobs_stub) { migration_stub(:Job) }
+
+  migration_context :up do
+    it "copies data from 'agent_id' to 'miq_server_id' column on jobs table" do
+      jobs_stub.create!(:name => "Hello Test Job", :agent_id => 111)
```

Comment:
I'd probably put the create/search criteria in a `let(:job_name) { "Hello Test Job" }` above and just use this job_name to create/find them.  Very minor, followup PR would be fine on both of these if no one else finds things to change for this PR.